terraform-ss-proxy/main.tf

```diff
@@ -0,0 +1,224 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+  required_version = ">= 1.0"
+  
+  # Local backend for state management
+  backend "local" {
```

Comment:
why not a S3 backend? As a more fault-tolerant option when our instances evaporate